frontend/src/Patient/PatientComponents/MedicationUploadPageComponents/MedicationCamera.jsx

```diff
@@ -19,18 +28,27 @@ function MedicationWebcam({ imgSrc, setImgSrc, setMedicationInformation, setIsLo
         })();
     }, [webcamRef]);
 
-    return (
+    function handleShowFailureAlert() {
+        setIsFailureAlertShowing(true);
+        setTimeout(() => setIsFailureAlertShowing(false), 3000);
+    }
```

Comment:
You can refactor your alert components a bit so this alert showing / timeout for hiding is setup by the child component itself. Non-blocking tho, we're only using it at 2 places